crates/evm/evm/src/executors/invariant/mod.rs

```diff
@@ -247,7 +249,17 @@ impl<'a> InvariantExecutor<'a> {
                     let mut state_changeset =
                         call_result.state_changeset.to_owned().expect("no changesets");
 
-                    collect_data(&mut state_changeset, sender, &call_result, &fuzz_state);
+                    if !&call_result.reverted {
```

Comment:
I don't think there's any valuable data we could collect from a reverted call, hence adding this, pls let me know if I am missing something.  (Further improvement when fail on revert set to false is to remove calls reverted from final sequence - should improve shrinking performance)

Reply:
:+1: , will follow up with a PR to exclude reverted call from final sequence if running with `fail-on-revert=false`, should improve shrinking phase a lot